tests/unit/monitor/api/report/test_report_parallel.py

```
@@ -0,0 +1,125 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from elementary.monitor.api.report.report import ReportAPI
+
+
+@pytest.fixture
+def mock_dbt_runner():
+    runner = MagicMock()
+    runner.project_dir = "/tmp/project"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/project"

(S108)